package.json

```diff
@@ -41,8 +40,9 @@
     "@types/node": "^6.14.7",
     "@typescript-eslint/eslint-plugin": "2.31.0",
     "@typescript-eslint/parser": "2.31.0",
+    "eslint": "6.8.0",
     "eslint-config-prettier": "6.11.0",
-    "prettier": "2.0.5",
+    "prettier": "1.19.1",
```

Comment:
why not prettier 2 if we're using node 10 already?

Reply:
the tests and the lib needs to work on Node 8, tests don't pass on 8 without downgrading